update_sc_name/update_scientific_name.py

```diff
@@ -58,20 +82,28 @@ def main():
                            default="production_processing", required=False)
     argparser.add_argument("--taxonomy", type=int, help="taxonomy id e.g. 9606", required=True)
     argparser.add_argument("--scientific-name", help=f"""new scientific name for taxonomy
-                                                     (put in "" for names having space in between)""", required=True)
+                                                     (put in "" for names having space in between)""", required=False)
```

Comment:
I am wondering if we should allow this at all... Any adjustment should belong in the normalization. Isn't it?

Reply:
If we want to be consistent with other sources like Ensembl, ENA and NCBI,  then we will have to keep these, as the names in those source also have single quotes. 
We just need to make sure, that whenever we use the scientific name for other things like naming a directory  we use the normalised one.

Reply:
removed the option